### PR TITLE
Design task/task 1

### PR DESCRIPTION
**Pull request by maeved**
basic functions of navigation bar and styling of background and colour schemes changed. Nav bar needs to be on all pages apart from login page.

### CODE REVIEW

File: kjourney/src/app/app.component.css [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
+/* body {
   background-image: linear-gradient(to bottom right, #61ab3f, white);
-  background-attachment: fixed;
-  background-size: cover;
-  overflow: hidden;
-  height: 100vh;
+  height: 100%;
   width: 100%;
-  font-family: sans-serif;
+} */
```
Comment (roisingoodman1) [inline kjourney/src/app/app.component.css]: why commented out? can this just be deleted?

File: kjourney/src/app/app.component.html [hunk 2]
```diff
@@ -4,22 +4,161 @@
...
+      }
+
+      .dropdown-content a:hover {
+        background-color: #ddd;
+      }
+
+      .dropdown:hover .dropdown-content {
+        display: block;
+      }
+    </style>
```
Comment (Conormc98) [inline kjourney/src/app/app.component.html]: Could we make this a separate css file?

**Review by roisingoodman1 — APPROVED** [review]

File: kjourney/src/app/app.component.html [hunk 3]
```diff
@@ -4,22 +4,161 @@
...
   </head>
   <body>
 
-  <header>
-    <h1></h1>
-    <img src="assets/logo.png" alt="kJourney Logo" display="inline-block" width="35%">
-  </header>
+    <header></header>
```
Comment (Conormc98) [inline kjourney/src/app/app.component.html]: If there's nothing in this header, do we need to keep it?

File: kjourney/src/app/app.component.html [hunk 4]
```diff
@@ -4,22 +4,161 @@
...
+        </div>
 
+        <router-outlet></router-outlet>
+      </div>
+    </div>
   </body>
 </html>
+
+<script>
+$(document).ready(function(){
```
Comment (Conormc98) [inline kjourney/src/app/app.component.html:84]: This is jquery code. Is jquery installed by npm, or will this need to be downloaded separately? Where is it imported?

**Review by Conormc98 — COMMENTED** [review]
Just a few questions 🙂

File: kjourney/src/app/app.component.css (hunk 1 above)
Comment (gordonkelly26) [reply kjourney/src/app/app.component.css]: sorry who are you?

Many thanks,
Gordon

**Review by gordonkelly26 — APPROVED** [review]
This is pretty neato. I agree with what Conor says.

Many thanks,
Gordon

File: kjourney/src/app/app.component.html (hunk 4 above)
Comment (maeved) [reply kjourney/src/app/app.component.html:84]: inserted jquery script, doesnt affect functionality for this sprint

**Review by Conormc98 — APPROVED** [review]

**Pull request merged**